src/walkthroughs/tdd/index.rst

```diff
@@ -21,3 +22,77 @@ Follow Along
 3. Write the tests
 4. Make them pass
 5. Clean up the code if there are any code smells
+
+Discuss Requirements
+____________________
+
+Following TDD and the red-green-refactor workflow we need to understand our requirements before writing any code.
+
+Today we will be implementing Range, and Trip Meter functionality.
+
+The Range functionality will calculate how many more miles the car can drive before running out of gas. The output should be a Double reprsenting the miles the car can drive before needing to fuel up.
+
+The Trip Meter will be a separate resetable odometer. This will allow a user to track the number of miles travelled on a specific trip.
+
+Create a Test List
+____________________
+
+Range
+
+* testInitialValue
+* testValueAfterDriving
+* testValueAtEmpty
+
+Trip Meter
+
+* testInitialValue
+* testAfterDriving
+* testResetAfterDriving
+* testResetThenDrive
+
+Write the Tests
+_______________
+
+Range
+
+.. image:: /_static/images/tdd/tdd-range-tests.png
```

Comment:
this is going to be annoying, but _'s in variable names go against the Java style that we use in the class. https://google.github.io/styleguide/javaguide.html#s5.1-identifier-names.  That is a pain because you would need to remake all of these images, if you agree that it's important enough to do

Reply:
I talked with Paul and we agreed we should do this. Paul was deep in another task, so I went ahead and replaced the images and variable references.